climada/hazard/trop_cyclone.py

```diff
@@ -1128,29 +1112,27 @@ def compute_angular_windspeeds(si_track, d_centr, close_centr_msk, model, cyclos
     return result
 
 def get_close_centroids(
-    t_lat: np.ndarray,
-    t_lon: np.ndarray,
+    si_track: xr.Dataset,
```

Comment:
The advantage of passing the whole track dataset instead of only the lat/lon coordinates is that the track dataset comes with an attribute  `"mid_lon"` that specifies the normalization of the longitudinal coordinates.